### PR TITLE
fix: restencil to fix e2e

### PR DESCRIPTION
**Pull request by chatchai-outreach**
## Jira ID

[XX-XX]

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,16 +1,15 @@
 {
   "license": "UNLICENSED",
   "devDependencies": {
-    "@outreach/semantic-release-circleci-orb": "^1.1.9",
```
Comment (jaredallard) [inline package.json:4]: Need to add this orb back

Comment (jaredallard) [reply package.json:4]: Make sure to run `yarn` after as well

**Review by jaredallard — APPROVED** [review]
LGTM after comment

File: package.json (hunk 1 above)
Comment (chatchai-outreach) [reply package.json:4]: should we update stencil-base for a custom block?

Comment (jaredallard) [reply package.json]: Yes, but `package.json` doesn't support blocks. I was going to write merging logic but still haven't got around to it :/

**Pull request merged**